src/sbvr-api/permissions.ts

```diff
@@ -1477,11 +1462,6 @@ export const resolveAuthHeader = async (
 	// TODO: Consider making tx the second argument in the next major
 	tx?: Tx,
 ): Promise<PermissionReq['apiKey']> => {
-	// TODO-MAJOR: remove this check
```

Comment:
I'm fine with it, but really curious about the reasoning :thinking: 
Was there a case that mid-request we decided we want to re-evaluate the cached value?

Reply:
This function (and the ones it calls) do not assign `req.apiKey` at all so it's weird for them to look at it's value and potentially refuse to check a different api key that has been passed because some external property has been set. In practice it works out because of how we have always used it thus far but it doesn't make sense in itself - it makes more sense for whatever assigns `req.apiKey` to check for it already existing if that is important